<?xml version="1.0" encoding="UTF-8"?>
<launch>

  <arg name="debug" default="false" />
  <arg name="full_map" default="false" />

  <param name="/node_expansion" type="bool" value="true" />
  <param name="/full_map_expansion" type="bool" value="$(arg full_map)" />

	<rosparam
	command="load"
	file= "$(find static_step_validation)/config/footsteps_talos.yaml"
	/>





<group if="$(eval arg('debug') == false)">
    <node 
	pkg="static_step_validation" 
	name="step_expansion_node" 
	type="step_expansion_node"
	output="screen"
	/>

</group>

    <node 
	pkg="tf2_ros"
	name="map_world_publisher"
	type="static_transform_publisher" 
	args=" 0 0 0 0 0 0 1 /odom /map"
	/>


</launch>
